tests/test_git_colocated.rs

```diff
@@ -268,10 +272,10 @@ fn test_git_colocated_fetch_deleted_or_moved_branch() {
     // 929e and 8d4e are abandoned, as the corresponding branches were deleted or
```

Comment:
This comment needs updating.

Reply:
Good catch. I replaced the two commit ids by "The two siblings of the working-copy commit".

Reply:
It might be better to refer to them by description (which you could change to make it one work for each commit). I think the working-copy commit has one more sibling that's not abandoned.